file-formats/aobj/aobj-v1.json

```diff
@@ -0,0 +1,144 @@
+          "enum": [
+            "standard",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "attributes": {
+      "title": "Attributes",
+      "description": "Attributes of an archiving object",
+      "type": "object",
+      "properties": {
+        "writeClass": {
+          "title": "Write Class",
+          "description": "Name of executable class or program which writes the data to the archive",
+          "type": "string",
+          "maxLength": 30
+        },
+        "deleteClass": {
+          "title": "Delete Class",
+          "description": "Name of executable class or program which deletes the data from the tables",
+          "type": "string",
+          "maxLength": 30
+        },
+        "reloadClass": {
+          "title": "Reload Class",
+          "description": "Name of executable class or program which reloads the data to the tables (may be empty)",
+          "type": "string",
+          "maxLength": 30
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "writeClass",
+        "deleteClass"
+      ]
+    },
+    "tablesToBeArchived": {
+      "title": "Tables to Be Archived",
+      "description": "Tables of business object instance from which data is archived.",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "title": "Table to Be Archived",
+        "description": "Tables of business object instance from which data is archived",
+        "type": "object",
+        "properties": {
+          "name": {
+            "title": "Name",
+            "description": "Name of table or structure",
+            "type": "string",
+            "maxLength": 30
+          }
+        },
+        "additionalProperties": false,
+        "required": [
+          "name"
+        ]
+      }
+    },
+    "archivingClasses": {
+      "title": "Archiving Classes",
+      "description": "Archiving classes are no ABAP classes, but IDs of reuse objects, whose data should be archived Together with the business object instance (for example change documents).",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "title": "Name of Archiving Class",
+        "description": "Archiving classes are no ABAP classes, but IDs of reuse objects, whose data should be archived Together with the business object instance (for example change documents).",
+        "type": "object",
+        "properties": {
+          "name": {
+            "title": "Name",
+            "description": "Name of archiving class",
+            "type": "string",
+            "maxLength": 30
```

Comment:
It seems the schema has not been updated, yet. Here archiving classes allow still 30 characters

@Olivia266 Would you update the JSON schema file, please?

@larshp I guess You would lie to ref #55 here :D

Reply:
@schneidermic0 agreement is signed. 
Json schema file updated.